python/ray/tests/test_advanced_2.py

```diff
@@ -745,6 +748,73 @@ def f():
     wait_for_pid_to_exit(pid1)
 
 
+def test_worker_leaked_when_drain(ray_start_regular):
+    @ray.remote
+    def hello():
+        return "hello"
+
+    # Warnup cluster
+    assert "hello" == ray.get(hello.remote())
+
+    driver_template = """
+import ray
+import os
+import ray
+import numpy as np
+import time
+
+ray.init(address="{address}", namespace="test")
+
+@ray.remote
+class KvStore:
```

Comment:
Could you simplify the driver script? I think the driver script is complicated and it makes me hard to get the key point of this test case.